AmericanMedicalAssociation/joe-style-guide
# Joe gallery

## JIRA Ticket(s)
N/A


## Description:
This is a temporary demo branch being put up for fast feedback. Most of these files are from JS libraries that will be pruned away for final release.
See full work description in the D8 branch https://github.com/AmericanMedicalAssociation/ama-d8/pull/2503


## To Test:
- pull and serve branch
- see further instructions in the D8 branch https://github.com/AmericanMedicalAssociation/ama-d8/pull/2503


## Automated Test:
N/A


## Style Guide:
N/A


## Relevant Screenshots/GIFs:
N/A


## Additional Notes:
Anything more to add?


---

[Pull Request Guidelines](https://github.com/palantirnet/development_documentation/blob/master/guidelines/pr_review.md)
